pytest.ini

```diff
@@ -1,11 +1,14 @@
 [pytest]
 addopts = --verbose -ra
-python_paths = src/ tests/
+pythonpath  = "src/ tests/"
```

Comment:
This is malformed.

```suggestion
pythonpath = src/ tests/
```

Reply:
Addressed here https://github.com/cisagov/con-pca-api/pull/632/commits/df00a92b6cb210c49deecd0ae5fb278750e253b4